examples/amp-story/animations-presets.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html amp>
+<head>
+  <meta charset="utf-8">
+  <title>amp-story</title>
+  <link rel="canonical" href="animations-presets.html">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <script async custom-element="amp-story" src="https://cdn.ampproject.org/v0/amp-story-0.1.js"></script>
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <style amp-custom>
+    body {
+      font-family: sans-serif;
+    }
+    h1 {
+      font-family: Menlo, Monaco, monospace;
+      font-weight: bold;
+      font-size: 24px;
+      width: 100%;
+      text-align: left;
+    }
+    amp-story-page {
+      background: #ddd;
+    }
+    amp-story-grid-layer {
+      justify-items: center;
+    }
+    .square,
+    .circle {
+      background: #4285f4;
+      width: 80px;
+      height: 80px;
+    }
+    .circle {
+      border-radius: 80px;
+    }
+    </style>
+</head>
+<body>
+  <amp-story standalone>
+    <amp-story-page id="fade-in-page">
+      <amp-story-grid-layer template="vertical">
+        <h1>fade-in</h1>
+        <div class="circle"
```

Comment:
Fade-in doesn't seem to work - do animations work for title pages?

Reply:
They're supposed to, but I think it's currently a bit buggy.